docs/modules/ROOT/pages/faq.adoc

```diff
@@ -88,7 +88,7 @@ For example, you can execute the below command on one member (i.e. listening on
 
 And you can execute the below command on the other member.
 
-`iperf -c` *`<IP address>`* `-d -p 5701`
+`iperf -c *``<IP address>``* -d -p 5701`
```

Comment:
these unrelated changes and the merge confict will probably go away if you fetch & merge main branch into your branch .